src/Apps/Feature/Components/FeatureHeader/FeatureHeaderFull.tsx

```diff
@@ -39,12 +39,11 @@ export const FeatureHeaderFull: React.FC<FeatureHeaderFullProps> = ({
             </Text>
 
             {subheadline && (
-              <Text
+              <HTML
                 variant={["md", "lg-display"]}
                 color="rgba(255, 255, 255, 0.8)"
-              >
-                {subheadline}
-              </Text>
+                html={subheadline}
+              />
             )}
```

Comment:
Simply brings `FeatureHeaderFull` in line with `FeatureHeaderDefault`, while preserving the text color:

https://github.com/artsy/force/blob/9c67452663d06f548714d68155be0f275b3b20ce/src/Apps/Feature/Components/FeatureHeader/FeatureHeaderDefault.tsx#L29-L35